packages/core/src/recon.ts

```diff
@@ -96,7 +97,7 @@ export class ReconApi extends Observable<ReconEventFeedResponse> implements IRec
    * @param initialCursor
    * @returns
    */
-  async init(initialCursor = 0): Promise<void> {
+  async init(initialCursor = '0'): Promise<void> {
```

Comment:
I might want to make a note that we should parse '0' on the server side as default if we change the shape someday. I wasn't expecting the client to send anything until receiving a response but this works fine. 

Just thinking.. you can close :)